pom.xml

```diff
@@ -104,6 +104,19 @@
         <version>2.11.0</version>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
```

Comment:
Can we have the version here. Otherwise you would be compromised with the stability of the release.

Reply:
<version>3.2.2</version> ?